Source/Core/Core/IOS/Network/IP/Top.cpp

```diff
@@ -1056,6 +1057,10 @@ IPCReply NetIPTopDevice::HandleGetAddressInfoRequest(const IOCtlVRequest& reques
   if (!request.in_vectors.empty() && request.in_vectors[0].size > 0)
   {
     nodeNameStr = memory.GetString(request.in_vectors[0].address, request.in_vectors[0].size);
+    if (std::optional<std::string> patch = WC24PatchEngine::GetNetworkPatch(nodeNameStr, false))
```

Comment:
For the sake of making call sites more readable, consider changing the second argument to an enum class:

```cpp
// in WC24PatchEngine
enum class IsKD : bool;

std::optional<std::string> GetNetworkPatch(..., IsKD is_kd);
```

a call to GetNetworkPatch would look like this:

```cpp
WC24PatchEngine::GetNetworkPatch(nodeNameStr, IsKD{false});
```

making it immediately obvious what the boolean represents